src/download.ts

```diff
@@ -88,13 +88,24 @@ export function getSkylinkUrl(this: SkynetClient, skylink: string, customOptions
   // TODO: fix subdomain + includePath
   let url;
   if (opts.subdomain) {
-    const skylinkPath = parseSkylink(skylink, { getPath: true });
-    skylink = parseSkylink(skylink);
+    // Get the path from the skylink.
+    const skylinkPath = parseSkylink(skylinkStr, { getPath: true });
```

Comment:
I would call `getPath` -> `onlyPath`, I actually had to look up what is it doing otherwise, wasn't really intuitive for me